support/carbon/carbon.go

```diff
@@ -7,27 +7,25 @@ import (
 )
 
 type Clock struct {
-	testNow  bool
-	testTime Carbon
 	timezone string
 }
 
+var testCarbon = carbon.NewCarbon()
```

Comment:
Why has this line?

Reply:
Because that's how You use this feature in carbon module.